v2/pkg/archive/image-blob-gatherer.go

```diff
@@ -0,0 +1,112 @@
+package archive
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/containers/image/v5/manifest"
+	"github.com/containers/image/v5/transports/alltransports"
+	"github.com/openshift/oc-mirror/v2/pkg/mirror"
+)
+
+type ImageBlobGatherer struct {
+	ctx  context.Context
+	opts *mirror.CopyOptions
+}
+
+func NewImageBlobGatherer(ctx context.Context, opts *mirror.CopyOptions) *ImageBlobGatherer {
+	return &ImageBlobGatherer{
+		ctx:  ctx,
+		opts: opts,
+	}
+}
+func (o *ImageBlobGatherer) GatherBlobs(imgRef string) (blobs []string, retErr error) {
+	blobs = []string{}
+	o.opts.DeprecatedTLSVerify.WarnIfUsed([]string{"--src-tls-verify", "--dest-tls-verify"})
+	// o.opts.All = true
+	o.opts.RemoveSignatures, _ = strconv.ParseBool("true")
+
+	if err := mirror.ReexecIfNecessaryForImages([]string{imgRef}...); err != nil {
+		return blobs, err
+	}
+
+	// policyContext, err := o.opts.Global.GetPolicyContext()
```

Comment:
Are we going to ignore signature verification ?

Reply:
I'm not sure it is important to verify signatures when collecting the blobs for images that we are mirroring.
That said, I think you bring up a good point for the signatures: do signatures need to be transported in the archives? I have no idea.
WDYT?